0000-interledger-protocol-4/0000-interledger-protocol-4.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Interledger Protocol V4 (ILPv4)
+draft: 1
+---
+
+# Interledger Protocol V4
+
+Interledger is a protocol for sending packets of money across different payment networks or ledgers. ILPv4 is a simplification of previous versions of the protocol that is optimized for routing large volumes of low-value packets, also known as "penny switching". ILPv4 can be integrated with any type of ledger, including those not built for interoperability, and it is designed be used with a variety of higher-level protocols that implement features ranging from quoting to sending larger amounts of value with chunked payments.
+
+## Overview
+
+### Design Goals
+
+- **Neutrality** - Interledger is not tied to any company, currency, or network.
+- **Interoperability** - ILP should be usable across any type of ledger, even those that were not built for interoperability.
+- **Security** - Senders, connectors, and receivers should be protected from one another and especially isolated from risks posed by parties that they are indirectly connected to. Connectors should not be able to steal money from senders, and senders should not be able to tie up too much of connectors' funds or otherwise interfere with their operation.
+- **Simplicity** - The core ILP should be as simple as possible. There is room for significant variability at lower and higher layers of the [Interledger stack](../0001-interledger-architecture/0001-interledger-architecture.md#interledger-protocol-suite), but the Interledger layer is the one part where widespread agreement is needed for interoperability. Simplifying the core minimizes what people need to agree upon and enables broader adoption.
+- **End-to-End Principle** - Inspired by the Internet, any features that do not need to be implemented by the core Interledger Protocol and network of connectors should be built into the edges of the network (the sender and receiver).
+
+### Terminology
+
+- **Account** - Two peers establish an account with one another to track the current obligations they hold with one another.
```

Comment:
This account definition reminds me of loro/nostro accounts, but surely is not. Could it be renamed totally to Link or Stream or something else? For fun: that could lead to uplink/downlink metaphor, which reflects connector and client distinction quite right. @emschwartz

Reply:
There are probably some similarities, but I'm not a real expert on nostro/vostro accounts.

Link is an interesting idea, and kind of funny because the layer below the internetworking layer in the old OSI model is called the Data Link layer. You establish a "Money Link" with your peer...